README.md

```diff
@@ -139,7 +139,7 @@ more Emacs-y.
    IMPORTANT: Whenever the toolchain updates, you have to reinstall
    rustfmt manually.
 
-2. You will need a C compiler and toolchain. On Linux, you can do
+2. You will need Clang and a toolchain. On Linux, you can do
```

Comment:
Wouldn't it be better to say that it requires both GCC and Clang? It is possible to build Remacs entirely with Clang, but the instructions in this readme don't give all of the details needed to do that.

Reply:
I agree, and thank you for responding. I'll edit the README again and push the commit, let me know what you think when I do.

And I apologize for the late response. I always happen to be away from my PC like right when people respond. :/

Reply:
No need to apologize; this is an inherently asynchronous medium.